code/modules/antagonists/blob/powers.dm

```diff
@@ -248,7 +248,7 @@
 
 /mob/camera/blob/verb/expand_blob_power()
 	set category = "Blob"
-	set name = "Expand/Attack Blob ([BLOB_SPREAD_COST])"
+	set name = "Expand/Attack Blob (4)"
```

Comment:
This kinda sucks(verb names need to be compile time constants, and must not use string interpolation like this does) but nobody will adjust this define, we don't even use blob so I suppose it's okay.